tests/test_typechecker.py

```diff
@@ -4664,6 +4664,10 @@ class Foo(typing.Generic[T]):
         # No exception.
         resolve_fw_decl(Foo)
 
+    # See: https://github.com/Stewori/pytypes/issues/35
+    def test_frozenset(self):
+        self.assertTrue(pytypes.is_of_type(frozenset({1, 2, 'a', None, 'b'}), typing.AbstractSet[typing.Union[str, int, None]]))
```

Comment:
I aim to keep line widths < 80 according to pep 8. I know that many parts of pytypes are far away from pep 8 compliance, but in general I try to converge it to pep 8 rather than further away from it....

This is just cosmetic, so I would not consider it a blocker for accepting this PR.

Reply:
I often ignore PEP8 a little in tests in my code.